src/js/disability-benefits/actions/index.js

```diff
@@ -45,6 +46,7 @@ export function getClaims(filter) {
       claims => {
         dispatch({ type: SET_CLAIMS, claims: claims.data, meta: claims.meta });
         dispatch({ type: FILTER_CLAIMS, filter });
+        dispatch({ type: SORT_CLAIMS });
```

Comment:
This kind of multiple dispatching is not ideal in Redux, since it causes the UI to re-render after each dispatch. It also leads to harder to debug code, since actions aren't as easily associated with external events. 

And in the React components it results in UI code that is more coupled to the application logic because it has to know the series of granular updates to the application state that needs to happen when a user takes some action.

Some of that's just preference, but I do think making actions corresponding to what the user is doing rather than specific state changes results in more cleanly separated UI and application logic.